nvflare/app_common/dex/dxo_exchanger.py

```diff
@@ -0,0 +1,115 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import os
+import time
+from typing import Optional
+
+from nvflare.apis.dxo import DXO
+from nvflare.fuel.utils.pipe.file_accessor import FileAccessor
+from nvflare.fuel.utils.pipe.file_pipe import FilePipe
+from nvflare.fuel.utils.pipe.pickle_file_accessor import PickleFileAccessor
+from nvflare.fuel.utils.pipe.pipe import Message
+from nvflare.fuel.utils.pipe.pipe_monitor import PipeMonitor, Topic
+
+
+class DXOExchanger:
```

Comment:
This should be called DXOFileExchanger based on the abstract class DXOExchanger, which should use Pipe object.

Reply:
thanks, changed.